backend/controllers/plans.js

```diff
@@ -0,0 +1,67 @@
+
+
+const Plan = require('../models/Plan')
+const Task = require('../models/Task')
+const User = require('../models/User')
+const mongoose = require('mongoose');
+const ObjectId = mongoose.Types.ObjectId;
+
+const {retErr} = require('../utils/index');
+const {checkInputs} = require('../utils/index')
+
+
+exports.findPlanToOwner = async (req, res) => {
+    let msgSender = req.user.userName;
+    let user = await User.findOne({userName: msgSender}, (err, data)=>{})
+    Plan.findOne( {_id: user.plan } ,(err, data) => {
+        if (!data)  return  retErr(res, err, 418, 'USER_NOT_IN_ANY_PLAN');
+        if (err) {
+            return  retErr(res, err, 418, 'DB_ERROR');
+        } else {
+            return res.status(200).json({data: data});
+        }
+    });
+}
+
+exports.findAllPlans = (req, res) => {
```

Comment:
Ich würde es `findAll` nennen, da du hier bereits bei `plans` bist. Oder noch besser  `index`, wie es in REST üblich ist. https://blog.usejournal.com/rails-and-the-seven-restful-routes-838e3e5e85d9